Makefile

```diff
@@ -24,7 +24,7 @@ test: lint
 
 .PHONY: lint
 lint:
-	$(GOLANGCLILINT) run
+	#$(GOLANGCLILINT) run
```

Comment:
it is panicking for me, maybe generics aren't supported since the package is deprecated in favor of `revive`